src/data-mgt/python/devices-tranformation/airqoApi.py

```diff
@@ -9,10 +9,16 @@
 class AirQoApi:
     def __init__(self):
         self.AIRQO_BASE_URL = os.getenv("AIRQO_BASE_URL")
-        self.AIRQO_API_KEY = os.getenv("AIRQO_API_KEY")
+        self.AIRQO_API_KEY = f"JWT {os.getenv('AIRQO_API_KEY')}"
```

Comment:
Pardon me @NoahNsimbe , seems I'm not getting the logic here! How shall we update the JWT token once it expires since we are get it from the `.env` variable?

Reply:
Thats a great idea @danielogen, did think about it. my focus was on them being util files that can be used by anybody when need arises. They save a person from doing manual work of searching for devices. I just added the authorization part but the main aim was to add more query parameters....Currently, whoever wants to use them will have to create a JWT token using their account.